staging/src/k8s.io/kubectl/pkg/cmd/apiresources/apiresources_test.go

```diff
@@ -303,6 +375,7 @@ bazzes   b            somegroup/v1   true         Baz
 	}
 
 	for _, tc := range testCases {
+		resources = []groupResource{} // clean the package level variable to avoid stacking up of resources
```

Comment:
I don't think this should be here.  The unit tests passed previously without it, so this would indicate a change in behavior.

There has to be another way to handle this without changing the test.

Reply:
Agree. Moved it to where you suggested earlier.